pages/linux/tcpick.md

```diff
@@ -0,0 +1,17 @@
+# tcpick
+
+> Packet sniffing and network traffic analysis tool.
+> It can capture and display TCP connections and data. It can also monitor network traffic on a specific interface, host, or port.
+> More information: <https://manned.org/tcpick.8>.
+
+- Capture traffic on a specific [i]nterface, port and host::
+
+`tcpick -i {{interface}} -C -h {{host}} -p {{port}}`
```

Comment:
```suggestion
`sudo tcpick -i {{interface}} -C -h {{host}} -p {{port}}`
```
Can we add `sudo` to the general example too, or is it unnecessary?

Reply:
Yes it can be added. It makes sense @kbdharun let me push those changes as well.